adafruit_wiznet5k/adafruit_wiznet5k_dns.py

```diff
@@ -3,14 +3,16 @@
 #
 # SPDX-License-Identifier: MIT
 
+# Each function only has one return statement. Pylint seems confused.
+# pylint: disable=inconsistent-return-statements
```

Comment:
I think we can get by without this pylint disable. I assume it was flagging something on your end based on your comment. 

I tried removing this and running pre-commit on my side and it still passes pylint.

Lets try removing this disable and see if it passes inside actions here on github. You may have a slightly different version of something that leads to different behavior. If you try removing this and it succeeds here on github but is still failing for you locally and you want help trying to get your local environment to match I can try to help you with that.

Reply:
I removed the Pylint disable and pushed the changes. All checks passed. Yay!